vendor/extensions/articles/db/seeds.rb

```diff
@@ -20,31 +20,33 @@
 
 Refinery::Articles::Article.destroy_all
 
-Refinery::Articles::Article.create(
+if Rails.env.downcase != "production"
```

Comment:
it's in a couple places but if you search of `.env.downcase` that's probably going to be thorough enough